package.json

```diff
@@ -89,7 +91,7 @@
     "elliptic": "^6.5.4",
     "js-sha3": "^0.8.0",
     "multiformats": "^9.4.10",
-    "uint8arrays": "^3.0.0"
+    "uint8arrays": "^3.0.0",
```

Comment:
Remove comma here. Causes yarn and npm JSON parsing error.

Reply:
Thanks for noticing this. I saw this too. package.json now parses.